lib/reply.js

```diff
@@ -43,10 +43,15 @@ Reply.prototype.send = function (payload) {
   if (payload && payload.isBoom) {
     this._req.log.error(payload)
     this.res.statusCode = payload.output.statusCode
+
+    for (var k in payload.output.headers) {
+      this.header(k, payload.output.headers[k])
+    }
```

Comment:
can you move this loop in its own top-level function? Also, use `Object.keys()` and a for loop, that is faster.